sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -2162,6 +2296,65 @@ public partial class EventDataInfo : System.ClientModel.Primitives.IJsonModel<Az
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.EventDataInfo>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.EventDataInfo>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class Exporter : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Exporter>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Exporter>
```

Comment:
Single word type name is not allowed, could we add some prefix to specify it?

Reply:
Renamed to `PipelineGroupExporter`